packages/element/src/itk-remote-viewport.ts

```diff
@@ -1,17 +1,18 @@
 /// <reference types="vite/client" />
 import { PropertyValues, css, html } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
+import { Ref, createRef, ref } from 'lit/directives/ref.js';
 import { SelectorController } from 'xstate-lit/dist/select-controller.js';
 
 import {
   RemoteActor,
   createHyphaActors,
   createRemoteViewport,
 } from '@itk-viewer/remote-viewport/remote-viewport.js';
+import { Image } from '@itk-viewer/remote-viewport/types.js';
```

Comment:
We want

```
import { Image } from 'itk-wasm'
```

The `Image` defined in *types.js* is not complete, has `ArrayBuffer` instead of `TypedArray`.

Reply:
Probably want `@itk-viewer/remote-viewport` package to export some `RemoteImage` type, which is just
```
import type { Image } from '@itk-wasm/htj2k'
export type RemoteImage = Image
```